dedicated-community-leader.md

```diff
@@ -0,0 +1,81 @@
+## Title
+
+_Dedicated Community Manager_
```

Comment:
If you change the title to Community Manger, you should highlight the importance of leadership to this role (somewhere in the pattern; perhaps context? Problem?)